### DISCUSSION TITLE
Equivalent to `git tag --contains <commit>`

### DISCUSSION THREAD

**Discussion by bytefluxio** [gitpython-developers/GitPython#1269] [Q&A] [answered]
What would be the gitpython equivalent of `git tag --contains <commit>`

I need to ascertain all tags that point to descendants of a commit. The git command is incredibly fast, but doing the following just isn't effective:
```
tags = []
  for tag in self.repo.tags:
      if self.repo.is_ancestor(git_hash, tag.commit):
          tags.append(tag)
```

**Comment by Byron** [comment, marked as answer]
I would go with `self.repo.git.tag(contains=commit)`.
